pymedphys/_mosaiq/helpers.py

```diff
@@ -479,4 +497,179 @@ def get_all_treatment_data(cursor, mrn):
+        )
+    mosaiq_fields["rx"] = rx
+
     return mosaiq_fields
+
+
+def get_staff_initials(cursor, staff_id):
+    initials = execute_sql(
+        cursor,
+        """
+        SELECT
+        Staff.Initials
+        FROM Staff
+        WHERE
+        Staff.Staff_ID = %(staff_id)s
+        """,
+        {"staff_id": staff_id},
+    )
+
+    return initials
+
+
+def get_treatment_history(cursor, mrn):
+    table = execute_sql(
+        cursor,
+        """
+        SELECT
+            Ident.IDA,
+            Patient.First_Name,
+            Patient.Last_Name,
+            Patient.Birth_DtTm,
+            Staff.Last_Name,
+            TxField.FLD_ID,
+            TxField.Field_Label,
+            TxField.Field_Name,
+            Site.Target,
+            Site.Rx_Depth,
+            Site.Target_Units,
+            Site.Technique,
+            Site.Modality,
+            TxFieldPoint.Energy,
+            Site.Dose_Tx,
+            Site.Dose_Ttl,
+            Site.Fractions,
+            Site.Frac_Pattern,
+            Site.Notes,
+            TxField.Version,
+            TxField.Meterset,
+            TxFieldPoint.Meterset_Rate,
+            TxField.Type_Enum,
+            TxFieldPoint.Gantry_Ang,
+            TxFieldPoint.Coll_Ang,
+            TxField.Ssd,
+            TxField.SAD,
+            Site.Site_Name,
+            TxField.Dyn_Wedge,
+            TxField.Wdg_Appl,
+            TxField.Block,
+            TxField.Blk_Desc,
+            TxField.Comp_Fda,
+            TxField.FDA_Desc,
+            TxField.Bolus,
+            SiteSetup.Isocenter_Position_X,
+            SiteSetup.Isocenter_Position_Y,
+            SiteSetup.Isocenter_Position_Z,
+            SiteSetup.Patient_Orient,
+            TxFieldPoint.Field_X,
+            TxFieldPoint.Coll_X1,
+            TxFieldPoint.Coll_X2,
+            TxFieldPoint.Field_Y,
+            TxFieldPoint.Coll_Y1,
+            TxFieldPoint.Coll_Y2,
+            TxFieldPoint.Couch_Vrt,
+            TxFieldPoint.Couch_Lat,
+            TxFieldPoint.Couch_Lng,
+            TxFieldPoint.Couch_Ang,
+            TxField.Tol_Tbl_ID,
+            TxField.BackupTimer,
+            SiteSetup.Status_Enum,
+            Site.Status_Enum,
+            TxField.IsHidden,
+            Site.Version,
+            Site.Create_ID
+
+        FROM Ident, TxField, Site, Patient, SiteSetup, TxFieldPoint, Staff
+        WHERE
+            TxField.Pat_ID1 = Ident.Pat_ID1 AND
+            TxField.Machine_ID_Staff_ID = Staff.Staff_ID AND
+            TxFieldPoint.FLD_ID = TxField.FLD_ID AND
+            TxFieldPoint.Point = 0 AND
+            Patient.Pat_ID1 = Ident.Pat_ID1 AND
+            SiteSetup.SIT_Set_ID = TxField.SIT_Set_ID AND
+            TxField.SIT_Set_ID = Site.SIT_Set_ID AND
+            Site.Version = 0 AND
+            SiteSetup.Version = 0 AND
+            Ident.IDA = %(patient_id)s
+        """,
+        {"patient_id": mrn},
+    )
+
+    mosaiq_fields = pd.DataFrame(
```

Comment:
Did you mean to use this variable?

Reply:
I'm not sure which you are asking about...

Reply:
`mosaiq_fields`

Reply:
Ah I was looking at the wrong spot. No, I did not mean to have that. That whole section was incomplete. Sorry about that.